tests/ztest/register/CMakeLists.txt

```
@@ -0,0 +1,10 @@
+# Copyright 2021 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
```

should be updated to `3.20.0`
```suggestion
cmake_minimum_required(VERSION 3.20.0)
```

---

Done